app/assets/stylesheets/application.scss

```diff
@@ -1,2 +1,276 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
+@import "font-awesome";
+
+$teal: #159E98;
+$other-highlight: #218578;
+$teal-grey: #32A69B;
+$dark-teal: #238479;
+$rust: #C18763;
+$yellow: #D6D8A6;
+
+$light-contrast: $teal-grey;
+$flash: #75E8E1;
+$highlight: $teal;
+$neutral: $yellow;
+
+/*$body: 'IM Fell DW Pica';*/
+$body: 'Tinos';
+$header: 'Tinos';
+
```

Comment:
Was this font used anywhere? Also it's a good idea to use a font stack, just in case the user is unable to load the font in their browser. For example, `'Tinos', Arial, sans-serif`